scannerlib/fileset/file_set.go

```diff
@@ -255,6 +255,38 @@ func walkProcessPaths(ctx context.Context, procName string, fileName string, tim
 		if procName != findProcName(string(stat)) {
 			continue
 		}
+
+		// If the optional argument "cli_arg_regex" is specified,
+		// run the relative regexp on the process cmdline content
+		if cliArgRegex != "" {
+			// Compile regex
+			compiledCliArgRegex, err := regexp.Compile("^" + cliArgRegex + "$");
+			if err != nil {
+				return fmt.Errorf("unable to compile cli arg regex \"%s\":\n%v", cliArgRegex, err)
```

Comment:
You can also use %q to produce double quoted strings: "unable to compile cli arg regex %q:\n%v"